src/mcts/search.cc

```diff
@@ -398,14 +398,16 @@ void Search::MaybeTriggerStop(const IterationStats& stats,
 // Return the evaluation of the actual best child, regardless of temperature
 // settings. This differs from GetBestMove, which does obey any temperature
 // settings. So, somethimes, they may return results of different moves.
-std::pair<float, float> Search::GetBestEval() const {
+std::pair<std::pair<float, float>, float> Search::GetBestEval() const {
```

Comment:
Maybe have `struct BestEval`? Or at least an `std::tuple` instead of nested pairs.
E.g.
```cpp
struct BestEval {
  float wl;
  float d;
  float ml;
};
```

Reply:
Done.